### ISSUE TITLE
Wrong typescript definitions for options colHeaders and rowHeaders

### ISSUE THREAD

**Issue by larvanitis** [handsontable/handsontable#5052]
### Description
The typescript definitions for `colHeaders` and `rowHeaders` have wrong function return type.
Currently they describe a function that returns nothing (`void`) but in reality it should return a string.

For example this line:
https://github.com/handsontable/handsontable/blob/7476f640754c49cd395d2c966a161540802c95b8/handsontable.d.ts#L1425

should be:
```
colHeaders?: ((index?: number) => string) | boolean | any[];
```

### Your environment
* Handsontable version: 2.0.0

**Comment by swistach** [comment]
Hi @larvanitis,
Thank you for your sharing. We will improve our definition file during next a few weeks, but you can also open a PR with your suggestion.

**Comment by aaronbeall** [comment]
Additionally `index` should not be an optional parameter.

**Comment by AMBudnik** [comment]
Hi @larvanitis 

we've released a new version yesterday - the `7.0.0` with a lot of crucial changes. Please update and share a comment if you still see that something's missing. Thank you.
